{"name": "test_oauthtest", "status": "broken", "statusDetails": {"message": "NameError: name 'url' is not defined", "trace": "def test_oauthtest():\n        tokenurl=\"http://thetestingworldapi.com/Token\"\n        data={'grant_type':'password','username':'pj222','password':'Shreeram@24'}\n        response=requests.post(tokenurl,data)\n>       res=requests.get(url, auth=HTTPBasicAuth(\"pj222\", \"Shreeram@24\"))\nE       NameError: name 'url' is not defined\n\nAutomationTests\\test_oauth.py:10: NameError"}, "start": 1709892006372, "stop": 1709892008044, "uuid": "05a6bc5f-e282-4f8d-aac3-5df5ce2065c1", "historyId": "dfd434c415bf77c0cab79655450768d2", "testCaseId": "dfd434c415bf77c0cab79655450768d2", "fullName": "AutomationTests.test_oauth#test_oauthtest", "labels": [{"name": "parentSuite", "value": "AutomationTests"}, {"name": "suite", "value": "test_oauth"}, {"name": "host", "value": "Sagar-PC"}, {"name": "thread", "value": "10516-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "AutomationTests.test_oauth"}]}